src/js/tabs/pid_tuning.js

```diff
@@ -1978,12 +1978,12 @@ TABS.pid_tuning.initialize = function (callback) {
             self.updating = true;
             Promise.resolve(true)
             .then(function () {
-                var promise;
                 if (semver.gte(FC.CONFIG.apiVersion, CONFIGURATOR.API_VERSION_MIN_SUPPORTED_PID_CONTROLLER_CHANGE) && semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_31)) {
                     FC.PID.controller = pidController_e.val();
-                    promise = MSP.promise(MSPCodes.MSP_SET_PID_CONTROLLER, mspHelper.crunch(MSPCodes.MSP_SET_PID_CONTROLLER));
+                    return MSP.promise(MSPCodes.MSP_SET_PID_CONTROLLER, mspHelper.crunch(MSPCodes.MSP_SET_PID_CONTROLLER));
                 }
-                return promise;
+                console.log(`Unsupported API version: ${FC.CONFIG.apiVersion}`);
+                return false;
```

Comment:
Why are we returning `false` here?

Reply:
Was returning promise in line 1983 only to see if API was supported. But yeah, this is about lexical scoping so removed. No sure if this was my idea or suggestion from Sonar ;-)